go/enclave/crosschain/message_bus_manager.go

```diff
@@ -199,7 +200,7 @@ func (m *MessageBusManager) CreateSyntheticTransactions(_ context.Context, messa
 		tx := &types.LegacyTx{
 			Nonce:    startingNonce + uint64(idx),
 			Value:    gethcommon.Big0,
-			Gas:      5_000_000,
+			Gas:      params.MaxTxGas,
```

Comment:
this might change between geth releases; at which point it can go in silently and old nodes wont be able to sync. Fine if we test this between releases.